chalk-macros/src/lib.rs

```diff
@@ -31,7 +31,7 @@ thread_local! {
 
 // When CHALK_DEBUG is enabled, we only allow this many frames of
 // nested processing, at which point we assume something has gone
-// awry.
```

Comment:
For what it's worth, "awry" is a valid English word, but it's probably worth changing it to something more understandable if it is causing confusion.

Reply:
need to have a dictionary at my hand.

Reply:
thanks for the info.

Reply:
away is not the right word.